cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,101 @@
 /// <reference types='cypress' />
 
-describe('Web Tables page', () => {
-  it('', () => {
+describe('Web Tables - Pagination Test', () => {
+  it('should change the page size and verify the number of rows', () => {
+    cy.visit('https://demoqa.com/webtables');
```

Comment:
if there are repeatable steps at the beginning of each test you can move them to the beforeEach

Reply:
Did it